crates/walrus-service/src/test_utils.rs

```diff
@@ -487,29 +486,30 @@ impl SimStorageNodeHandle {
                 Duration::from_millis(100),
             ))
         } else {
-            let processor_config = EventProcessorRuntimeConfig {
-                rpc_addresses: combine_rpc_urls(
-                    &sui_config.rpc,
-                    &sui_config.additional_rpc_endpoints,
-                ),
-                event_polling_interval: Duration::from_millis(100),
-                db_path: nondeterministic!(
-                    tempfile::tempdir()
-                        .expect("temporary directory creation must succeed")
-                        .path()
-                        .to_path_buf()
-                ),
-                rpc_fallback_config: None,
-                db_config: DatabaseConfig::default(),
-            };
-            let system_config = crate::node::events::event_processor::SystemConfig {
+            let processor_config =
+                crate::event::event_processor::config::EventProcessorRuntimeConfig {
```

Comment:
Question (nit): Is there a reason to qualify this instead of importing as we do normally (also the `SystemConfig` below)?

Reply:
ok will fix